lib/change_health/version.rb

```diff
@@ -1,3 +1,3 @@
 module ChangeHealth
-  VERSION = '5.3.1'.freeze
+  VERSION = '5.3.2'.freeze
```

Comment:
I believe while the change is small, it is to a public interface, right? Meaning other consumers of this gem may have set/used the service date properties on `InstitutionalService` model and would now get an error when attempting to do so (not backwards compatible)?
I do know that either way, the version will need to be updated as there has been a more recent published version than 5.3.1.